src/program-editors.rst

```diff
@@ -39,18 +39,20 @@ TODO: work on version control systems
 Accessibility
 =============
 
-Introduction and Overview
+Overview
 ----------------------------
 
-Accessibility in computing refers to the development and design of both software and hardware systems to be usable by people with disabilities. Broadly, "Providing accessibility means removing barriers that prevent people with disabilities from participating in substantial life activities, including the use of services, products, and information." :cite:`bergman1995towards`. This includes providing screen readers for those with visual impairments and alternative input devices for those with motor impairments. 
-
-Additionally, advances in computing technology over the previous decades have enabled richer communication for individuals with impairments. For example, augmentative and alternative communication (AAC) tools such as video calling or speech generating devices assist individuals with impairments in producing or comprehending speech.
+Accessibility in computing refers to the development and design of both software and hardware systems to be usable by people with disabilities. Many of the techniques used to create accessible software are also relevant to program editors, however, these present some challenges. For example, programming environements may include real-time debug information, use color to encode meaning, and contain large blocks of difficult to navigate text. These are just a few of many components which require special consideration in creating accessible program editors.
```

Comment:
"Many of the..." -- this is an awkwardly phrased sentence with the "however, " part, should be two sentences